src/Pages/InspirationalQuotesPage/InspirationalQuotesPage.tsx

```diff
@@ -2,23 +2,106 @@
 import {Quotecard} from "../../Components/Quotecard/Quotecard";
 import "./InspirationalQuotesPage.scss"
 
-export function InspirationalQuotesPage(){
-    return(
+export function InspirationalQuotesPage() {
+
+    const quotes = [
```

Comment:
Nice.

To make this even nicer we can do 2 things

1) move it outside of the `function` (at the moment we are re-declaring this array every time we re-render, which slightly wasteful (although computers are fast, so we probably get away with it)

2) give it a type! We can make an interface for these quotes, and then the type system can help us out! :)

Reply:
Thanks, Mike. 
Changes made.